packages/button/Button.test.tsx

```diff
@@ -1,45 +1,64 @@
 import * as expect from 'expect';
-import { h } from '../_helpers';
-import { mount } from 'bore';
+import { h, mount } from 'bore';
+import { emit } from 'skatejs';
 
 import { Button } from './index';
 
 describe( Button.is, () => {
 
-describe( `Custom element`, () => {
-  it( `should be registered`, () => {
-    expect( customElements.get( Button.is ) ).toBe( Button );
-  });
-  it( `should render via JSX IntrinsicElement`, () => {
-    return mount( <bl-button>Hello</bl-button> ).wait( (element) => {
-      expect( element.node.localName ).toBe( Button.is );
+  describe( `Custom element`, () => {
+
+    it( `should be registered`, () => {
+      expect( customElements.get( Button.is ) ).toBe( Button );
     });
-  });
-  it( `should render`, () => {
-    return mount( <Button /> ).wait(( element ) => {
-      expect( element.has( '.c-button' ) ).toBe( true );
+
+    it( `should render via JSX IntrinsicElement`, () => {
+      return mount( <bl-button>Hello</bl-button> ).wait(( element ) => {
+        expect( element.node.localName ).toBe( Button.is );
+      });
+    });
+
+    it( `should render`, () => {
+      return mount( <Button /> ).wait(( element ) => {
```

Comment:
don't remember either ¯\_(ツ)_/¯ 

maybe few suggestions about your review attitude:
- if you intend to stop merging things because some unrelated unanswered questions, that's very unfortunate for velocity and for everyone that's participating on this project
- we can change it ofc in another PR which should refactor other tests that have the same pattern and update the docs accordingly